src/processes/follow-topic-process.js

```diff
@@ -79,10 +112,21 @@ const sendMessageAsAnonymous = async (serverClient, communityMessageFormat, data
   });
 
   // if (channelState.messages.length === 0) {
-  await newChannel.sendMessage({
-    text: communityMessageFormat.message,
-    user_id: communityMessageFormat.user_id,
-    is_auto_message: true
+  const serverMessage = await newChannel.sendMessage(
+    {
+      text: communityMessageFormat.message,
+      user_id: communityMessageFormat.user_id,
+      is_auto_message: true
+    },
+    {skip_push: true}
+  );
+
+  sendPushNotification({
+    serverChannel: newChannel,
+    serverMessage,
+    targetUserId: data?.user_id,
+    senderUsername: `${channelInfo?.anon_user_info_color_name} ${channelInfo?.anon_user_info_emoji_name}`,
+    text: communityMessageFormat?.message
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `sendMessage` and `sendPushNotification` functions are not awaited, which means they're executed asynchronously. If these functions throw an error, it won't be caught by the surrounding try-catch block. Consider using `await` to ensure any errors are properly handled.
```diff
-  const serverMessage = await newChannel.sendMessage(
+  const serverMessage = await newChannel.sendMessage(
     {
       text: communityMessageFormat.message,
       user_id: communityMessageFormat.user_id,
       is_auto_message: true
     },
     {skip_push: true}
   );
 
-  sendPushNotification({
+  await sendPushNotification({
     serverChannel: newChannel,
     serverMessage,
     targetUserId: data?.user_id,
     senderUsername: `${channelInfo?.anon_user_info_color_name} ${channelInfo?.anon_user_info_emoji_name}`,
     text: communityMessageFormat?.message
   });
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->